datafusion/substrait/src/serializer.rs

```diff
@@ -56,8 +67,9 @@ pub async fn deserialize(path: &str) -> Result<Box<Plan>> {
     deserialize_bytes(protobuf_in).await
 }
 
+/// Deserializes a plan from the bytes.
```

Comment:
Follow-up suggestion (breaking change): this function is marked `async` but it doesn't have to be.